mitiq/zne/tests/test_zne.py

```diff
@@ -94,21 +94,23 @@ def executor(circuit) -> float:
 def test_with_observable_batched_factory(executor):
```

Comment:
you are not using the `executor` argument anywhere in the test, which means you are not using all the parameter values 🤔

Reply:
Good catch! That was left over from testing and never got changed back. Fixed in https://github.com/unitaryfund/mitiq/pull/2602/commits/52765b73bd60da44f267d6fa2c96197841a10aa2.